packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2717,14 +2717,8 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
    * @param localAudioTrack -.
    */
   public mute = (localAudioStream: LocalMicrophoneStream): void => {
-    const localAudioTrack = localAudioStream.outputStream.getAudioTracks()[0];
-    if (this.muted) {
-      localAudioTrack.enabled = true;
-      this.muted = false;
-    } else {
-      localAudioTrack.enabled = false;
-      this.muted = true;
-    }
+    localAudioStream.setUserMuted(!this.muted);
```

Comment:
Should we add a check here to see if localAudioStream is passed or not? I remember in one of the contact center scenarios they were calling the answer method without localAudioStream, since they were using a js file it didn't show as an error. So a similar scenario might come here also.

Reply:
@Shreyas281299 localAudioStream is a mandatory parameter for dial, answer and mute functions. So expectation here is they should always be present and specifically for mute, if there is no localAudiostream present then call won't exist and hence mute function is not applicable for such cases. Do you still feel we should add the check for stream here?